Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "You know, I recently read about swimming. There are so many species.  But I was attracted the most by scuba diving.
```

Comment:
Species of what? 

I think bringing up that she recently read about swimming is a bit random, especially when you suddenly connect it to scuba diving without elaborating. 

Instead of just reading up about swimming, what about having Monika say that she was thinking about water sports she could do with you, and decided to look into scuba diving?